oak_remote_attestation_verification/proto/verification.proto

```diff
@@ -0,0 +1,47 @@
+//
+// Copyright 2023 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+syntax = "proto3";
+
+package oak.verification.v1;
+
+enum TeePlatform {
+  UNSPECIFIED = 0;
+  ANY = 1;
```

Comment:
What is the expected behavior when this option is specified? Perhaps add a comment to clarify.